### ISSUE TITLE
Sons of Hodir rep questline bug

### ISSUE THREAD

**Issue by Imisstheoldkanye** [dalaranwow/dalaran-wow#4894]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I cannot complete the quest The Drakkensryd due a bug preventing Hyldsmeet Drakeriders from being counted towards the quest. This means that I am unable to earn any Sons of Hodir rep, which is crucial for DPS specs. From what I've been told the problem is already known.
**Expected behaviour**: Tell us what should happen instead
I should be able to kill Hyldsmeet Drakeriders and have it count towards the quest, so that I can finish the quest line and get rep with the Sons of Hodir.
**Steps to reproduce the problem**:

1. xxxxxxxxxx
2. 
3. 

**Include proofs for this behaviour**
xxxxxxxxxxxxx
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.
xxxxxxxxxxxxxxxx
**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Account name is Chefdynamic and this is on the toon Imisstheoldk.
You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Crispybuilt** [comment]
Is this on Kirin Tor realm?

**Comment by TeridanDW** [comment]
Dublicate with #4349
